examples/README.md

```diff
@@ -186,9 +186,24 @@ identities. An additional per-repository default policy can be specified for
 identities not in the whitelist. Furthermore, a global admin policy can also be
 specified which can override per-repository policies.
 
+Regexes can also be used as repository paths.
+
+Authorization is granted based on the longest path matched.
+For example repos2/repo repository will match both ".*" and "repos2/repo" keys,
+in such case repos2/repo policy will be used.
+
 ```
 "accessControl": {
-    "repos1/repo": {
+    ".*": {
```

Comment:
Can we use the same pattern we use for "sync" feature?
\* and ** for subdirs?

Reply:
Added glob patterns